frontend/src/scenes/project-homepage/projectHomepageLogic.tsx

```diff
@@ -1,30 +1,32 @@
-import { kea } from 'kea'
+import { afterMount, connect, kea, path, selectors } from 'kea'
 
 import { projectHomepageLogicType } from './projectHomepageLogicType'
 import { teamLogic } from 'scenes/teamLogic'
 import { dashboardLogic } from 'scenes/dashboard/dashboardLogic'
 import { DashboardPlacement, InsightModel, PersonType } from '~/types'
 import api from 'lib/api'
+import { subscriptions } from 'kea-subscriptions'
+import { loaders } from 'kea-loaders'
 
-export const projectHomepageLogic = kea<projectHomepageLogicType>({
-    path: ['scenes', 'project-homepage', 'projectHomepageLogic'],
-    connect: {
+export const projectHomepageLogic = kea<projectHomepageLogicType>([
+    path(['scenes', 'project-homepage', 'projectHomepageLogic']),
+    connect({
         values: [teamLogic, ['currentTeamId']],
-    },
+    }),
 
-    selectors: {
+    selectors({
         primaryDashboardId: [() => [teamLogic.selectors.currentTeam], (currentTeam) => currentTeam?.primary_dashboard],
         dashboardLogic: [
             (s) => [s.primaryDashboardId],
             (primaryDashboardId): ReturnType<typeof dashboardLogic.build> | null =>
-                dashboardLogic.build(
-                    { id: primaryDashboardId ?? undefined, placement: DashboardPlacement.ProjectHomepage },
-                    false
-                ),
+                dashboardLogic({
```

Comment:
What's the difference between straight-up initialization and `.build()`?

Reply:
It's literally the same thing. https://github.com/keajs/kea/blob/kea-3.0/src/kea/kea.ts#L78-L83